### PR TITLE
Update diagrams

### PR DESCRIPTION
**Pull request by roldanbob**
Add the following reworked image files:

OLD			|        NEW		|	       CALLING FILE
constraint.jpg		constraint.png 	        r_ddl-metadata-for-schema-objects.adoc
create_column.jpg	create_column.png	        r_ddl-metadata-for-schema-objects.adoc
create_trigger.jpg	create_trigger.png	        r_ddl-metadata-for-schema-objects.adoc
create_view.jpg	create_view.png		r_ddl-metadata-for-schema-objects.adoc
create_procedure.jpg	procedure.png		r_ddl-metadata-for-schema-objects.adoc
teiid-architecture.png	data-virt_arch.png	as_architecture.adoc
query_plan_2.png	   query-plan.png) 	        r_planning-overview.adoc
teiid-query-plan-3.png	query-plan-2.png) 	r_query-planner.adoc:
vdb.png	                        virtual_db.png		as_virtual-databases.adoc

The following files were also re-done, but I could not find references to them:  
alter_Column.png  create_table.png  options_1.png  query_1.png
alter.png         example_2.png     options_2.png

### CODE REVIEW

File: reference/as_architecture.adoc [hunk 1]
```diff
@@ -22,9 +22,9 @@ used to connect/federate/transform datasources to produce a `.vdb` file.
 * **quickstarts** - A maven quickstart showing how to utilize {{ book.productnameFull }}.
 {% endif %}
 
-{% if book.targetWildfly %}
-image:images/teiid-architecture.png[images/teiid-architecture.png]
-{% endif %}
+
+image:images/data-virt-arch.png[images/data-virt-arch.png]
+
```
Comment (shawkins) [inline reference/as_architecture.adoc:27]: The new image isn't very applicable to wildfly - especially the operator.  While it is possible to containerize Teiid with Wildfly, we aren't emphasizing that at all.  We can conditionalize the new to only spring boot.

Comment (roldanbob) [reply reference/as_architecture.adoc:27]: Should we retain the old version of the diagram (teiid-architecture.png) for Wildfly? There's also a DV Engine 
data_virt_engine diagram yet to be inserted into the Architecture topic. Should that likewise be conditionalized to SB only?

Comment (rareddy) [reply reference/as_architecture.adoc:27]: I believe old one has few more boxes in it, So I leave as is for Wildfly with old diagram

Comment (shawkins) [reply reference/as_architecture.adoc:27]: The consensus is to leave the old diagram for Wildfly.  

The DV engine image is not specific to spring boot, but probably calls netty out too prominently.  There are effectively 2 client transports - our netty sockets and the web endpoints (which can be odata or rest).

Comment (roldanbob) [reply reference/as_architecture.adoc:27]: ok, let me close this pr and resubmit without deleting the old version. I'll also implement the conditionalization as discussed, and add the new DV Engine diagram.
@rareddy is there any specific context that I should provide in introducing the DV engine diagram, or should I essentially say, _hey, here's a more detailed view of the internals of the DV service_?

Comment (rareddy) [reply reference/as_architecture.adoc:27]: IMO, no additional context is needed

**Pull request closed without merging**